actors/market/src/lib.rs

```diff
@@ -538,9 +538,8 @@ impl Actor {
 
         Ok(VerifyDealsForActivationReturn { sectors: sectors_data })
     }
-    /// Activate a set of deals, returning the combined deal weights.
-    /// The weight is defined as the sum, over all deals in the set, of the product of deal size
-    /// and duration.
+    /// Activate a set of deals, returning the combined deal spaces and extra info for verified deals.
+    /// The space is defined as the sum of deal size in bytes over all deals in the set
```

Comment:
The second line of this comment might be confusing now with changes to the return value. I think we can just drop it.